Rename Gulf of Mexico to Gulf of America

KateFriedman-NOAA opened the pull request:
# Description

This PR updates the name of the "Gulf of _Mexico_" to the "Gulf of _America_". This includes fix/parm file updates and a new UWM tag for WW3 rename updates.

Refs #3326

# Type of change

Production update

# Change characteristics

- Fix and parm file updates
- UWM tag update

KateFriedman-NOAA committed:
Rename Gulf of Mexico to Gulf of America

Refs #3326

KateFriedman-NOAA committed:
Update UWM tag to GFS.v16.3.22

Refs #3326

DavidHuber-NOAA reviewed (approved):
lgtm

JessicaMeixner-NOAA commented:
Should we make a comment about the WW3/model changes being in a comment - just so people know that there's no expected impact to answers, etc?

KateFriedman-NOAA commented:
> Should we make a comment about the WW3/model changes being in a comment - just so people know that there's no expected impact to answers, etc?

Yes, good idea, I'll update the release notes to make a note that results are not changed from this.

KateFriedman-NOAA committed:
Update release notes

Add language to release notes to make sure it's understood
that this is a non-science change and does not impact model results

Refs #3326

KateFriedman-NOAA commented:
Committed release notes update @ 698aabba

JessicaMeixner-NOAA reviewed (approved):
Thanks @KateFriedman-NOAA  for facilitating these changes.
